airbyte-integrations/connectors/source-zendesk-support-singer/Dockerfile

```diff
@@ -5,11 +5,12 @@ RUN apt-get update && apt-get install -y bash && rm -rf /var/lib/apt/lists/*
 ENV CODE_PATH="source_zendesk_support_singer"
 ENV AIRBYTE_IMPL_MODULE="source_zendesk_support_singer"
 ENV AIRBYTE_IMPL_PATH="SourceZendeskSupportSinger"
+ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/source.py"
```

Comment:
```suggestion
ENV AIRBYTE_ENTRYPOINT "/airbyte/base.sh"
```

Reply:
`docker inspect airbyte/source-zendesk-support-singer:0.2.2` shows `/airbyte/base.sh` as the existing entrypoint.